docs/samples/fluid/kserve-huggingfaceserver.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: serving.kserve.io/v1alpha1
+kind: ClusterServingRuntime
+metadata:
+  name: kserve-huggingfaceserver
```

Comment:
This name is already used by the huggingface runtime installed by default, we can allow more customizations from the out-of-the-shelf one so you do not need to create another huggingface runtime with slight differences.

https://github.com/kserve/kserve/blob/e879a397b38fdcbebd8ec004c4da87916fb30a03/charts/kserve-resources/templates/clusterservingruntimes.yaml#L399

Reply:
Hi @yuzisun , yes. Here I patched the existing one. 

In this case, I can have a clusterservingruntime called `custom-kserve-huggingfaceserver`. Do you think I should use it via `predictor.model.runtime: custom-kserve-huggingfaceserver` or set `priority: 0` for `huggingface` model format in the custom runtime so that the controller will pick it up first for `huggingface` format?